.eslintrc 2.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
```

Comment:
What is the use of file when we already have an .eslintrc.js file

Reply:
will delete right away